modules/apps/layout/layout-page-template-admin-web/src/main/java/com/liferay/layout/page/template/admin/web/internal/exporter/PortletConfigurationExporterRegistry.java

```diff
@@ -27,8 +27,8 @@
 /**
  * @author Jürgen Kappler
  */
-@Component(service = PortletConfigurationExporterTracker.class)
-public class PortletConfigurationExporterTracker {
+@Component(service = PortletConfigurationExporterRegistry.class)
+public class PortletConfigurationExporterRegistry {
 
 	public PortletConfigurationExporter getPortletConfigurationExporter(
 		String portletName) {
```

Comment:
@JanisZhang, no usages use this. I think we can remove it. You may add the reason in the commit message. For example, its usage was removed by the commit 80343385f95dae5c75995cd2b53003f8f3617f18 WidgetInstanceDTOConverter class. Otherwise, it looks a little weird. Other commits are the first is **Renam..**, the second is **Apply usages**

Reply:
@yuhai Got it, Thanks!